package.json

```diff
@@ -45,6 +45,8 @@
     "pino": "^5.13.6",
     "pino-pretty": "^3.3.0",
     "sentiment": "^5.0.2",
+    "spam-filter": "^1.1.1",
```

Comment:
Don't hand edit this file for dependencies.  Prefer `npm install --save package` so it puts it in the right place (npm will alphabetize it).

Reply:
Github was saying there was a conflict in the package.json, so I edited it to include all the dependencies

Reply:
I'm getting the same conflicts again.
Is there any way to resolve them without manually editing package.json?

Reply:
Never mind, the PR got merged while I was typing that